flutter/lib/src/sentry_flutter.dart

```diff
@@ -230,6 +230,8 @@ mixin SentryFlutter {
   /// Manually set when your app finished startup. Make sure to set
   /// [SentryFlutterOptions.autoAppStart] to false on init. The timeout duration
   /// for this to work is 10 seconds.
+  @Deprecated(
+      'Will be removed in v9. This functionality will not be supported anymore.')
```

Comment:
If it will be removed in v9, we should remove it. Or update the text to refrlect that it is still available in v9, but will be reomved later, no?